src/modules/auth/user.ts

```diff
@@ -0,0 +1,7 @@
+export type User = {
+  kid: string;
```

Comment:
Faz sentido uma string ou deveria ser um dado booleano? Se for booleano, nao seria melhor a nomenclatura isKid?